### PR TITLE
[af] Machine translation of missing keys

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

**Comment by caouecs** [comment]
@JacquesvanWyk can you help us and review this automatic pull request, please ? thanks by advance

File: locales/af/validation-inline.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
     'alpha_dash'           => 'Hierdie veld mag slegs letters, syfers, strepies en onderstrepe bevat.',
     'alpha_num'            => 'Hierdie veld mag slegs letters en syfers bevat.',
     'array'                => 'Hierdie veld moet \'n array wees.',
-    'attached'             => 'This field is already attached.',
+    'attached'             => 'Hierdie gebied is reeds aangeheg.',
```
Comment (JacquesvanWyk) [inline locales/af/validation-inline.php:23]: Just use the word "veld" instead of "gebied" here

**Comment by JacquesvanWyk** [comment]
Just the one change and the rest looks good.

**Comment by caouecs** [comment]
Ok, thank you, I merge, and I fix :)

**Pull request merged**